### PR TITLE
ENDOC-457 7PNG update for Page Management Tutorial

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**

### CODE REVIEW

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 1]
```diff
@@ -1,82 +1,98 @@
 
 # Page Management
 
-The App Builder provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to layout the available frames on a page.
+The [Entando App Builder](../../docs/compose/app-builder.md) provides the capability to publish application pages containing content as well as other Entando components. Page Templates are used to define the page layout of available frames.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:4]: Page Templates are used to define the page layout of available frames. >> don't templates define how the frames are laid out on a page, seems strange the other way around??

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 2]
```diff
@@ -1,82 +1,98 @@
...
-   - To add a frame in a specific place of the page, add `<@wp.show frame=0 />`, where frame is the pos variable from the `JSON configuration`. `<#assign wp=JspTaglibs["/aps-core"]>` is required at the top of the template to setup the `wp` variable.
-   - Common code can be shared across pages by using [UX Fragments](./widgets-fragments.md#create-a-ux-fragment) and ```<@wp.fragment code="\<FRAGMENT\_CODE\>" escapeXml=false /\>```.
+- A `JSON configuration` field which lists the set of frames that can be used on a page. 
+- A `Template` field which uses Freemarker code to setup the HTML page itself.
+
+### JSON Configuration
+
+Each item in the `frames` array of the JSON configuration represents a frame or slot in the page, characterized by the following values:
+- `pos`: A zero-based position index (starts from zero). This value is used by APIs to address a specific widget on the page.
+- `descr`: The frame description displayed in the Page Design view.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:72]: Should Page Design or Page Template be backticked? I'm running into this and am not sure

File: vuepress/docs/next/tutorials/compose/page-management.md (hunk 1 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/page-management.md:4]: yes... the page layout of the frames = the layout of the frames on a page, right?? i can switch back, it was just a bit wordy

File: vuepress/docs/next/tutorials/compose/page-management.md (hunk 2 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/page-management.md:72]: no, i don't think so... we haven't adopted that convention and i don't think it's necessary; we already have a lot of backticking going on and to do so would dilute the significance of where it's really applicable. what do you think?

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 3]
```diff
@@ -117,10 +132,11 @@ This tutorial prepares a Page Template with two frames on it.
...
-Include the following fragment in the `head` section if you want to make use of the user's identity information from Keycloak.
+Include the following fragment in the `head` section to make use of the user's Keycloak identity information.
 ```ftl
 <@wp.fragment code="keycloak_auth" escapeXml=false />
 ``` 
 
-3. You should see the `Template preview` reflecting the desired two frame layout.
-4. Click `Save` to save your template. 
\ No newline at end of file
+**3. Verify the `Template preview` is reflecting the desired two frame layout.**
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:140]: Verify **that** the `Template preview`...

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 4]
```diff
@@ -1,82 +1,98 @@
...
 
--   Click the green **Save and Design** button. This brings you to the Designer section which lets you place widgets into the Page Template frames.
+![image](./img/publish_page3.png)
 
-![image](./img/Publish3.png)
+**4. Add a widget.**
 
--   Add widgets by dragging them from the right column into the desired frame. 
--   If you select the **Content** widget, its corresponding settings page will be shown.
+This is done by dragging a widget from the right panel into the desired frame. Adding the Content widget will load the configuration page shown below.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:33]: I think this Content needs to be backticked

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 5]
```diff
@@ -1,82 +1,98 @@
...
 
-![image](./img/Publish4.png)
+![image](./img/publish_page4.png)
 
--   Click **Add existing content** to bring up the Content selection view.
+**5. Click `Add existing content`.**
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:37]: `Add existing content`>> should existing content be in caps?

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 6]
```diff
@@ -1,82 +1,98 @@
...
+You can search for the name of the content that you would like to publish. Remember that content must be saved and approved before it can be published to a page.
 
--   Press the **Save** button.
+**6. Select the desired content item(s) from the list.** 
 
-![image](./img/Publish5b.png)
+**7. Click the `Continue` button.**
 
--   The title of the selected content item will now be displayed in the widget settings.
+The selected content item will now be displayed.
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:49]: Content should be backticked & capitalized?

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 7]
```diff
@@ -1,82 +1,98 @@
...
 
--   Press the **Save** button.
+![image](./img/publish_page5b.png)
 
--   Click on the **Preview** button to show the page preview with the updated settings.
+**8. Click the **Save** button.**
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:53]: Save should be backticked

File: vuepress/docs/next/tutorials/compose/page-management.md [hunk 8]
```diff
@@ -1,82 +1,98 @@
...
-      - `sketch` - an object with 4 coordinates (x1,x2,y1,y2) to allow the developer to place the widgets in the Page preview. Sketch’s x and y values go from 0 to 11 (similar to columns in Bootstrap), so if you want to place a 2x2 frame at the top left corner of the page, the values would be `x1: 0, x2: 1 y1: 0 y2: 1`.
-2. A `Template` field which uses Freemarker code to setup the HTML page itself.
-   - To add a frame in a specific place of the page, add `<@wp.show frame=0 />`, where frame is the pos variable from the `JSON configuration`. `<#assign wp=JspTaglibs["/aps-core"]>` is required at the top of the template to setup the `wp` variable.
-   - Common code can be shared across pages by using [UX Fragments](./widgets-fragments.md#create-a-ux-fragment) and ```<@wp.fragment code="\<FRAGMENT\_CODE\>" escapeXml=false /\>```.
+- A `JSON configuration` field which lists the set of frames that can be used on a page. 
+- A `Template` field which uses Freemarker code to setup the HTML page itself.
+
+### JSON Configuration
+
+Each item in the `frames` array of the JSON configuration represents a frame or slot in the page, characterized by the following values:
```
Comment (jyunmitch) [inline vuepress/docs/next/tutorials/compose/page-management.md:70]: frame> Frame?

File: vuepress/docs/next/tutorials/compose/page-management.md (hunk 5 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/page-management.md:37]: maybe, but that's not how it's displayed

File: vuepress/docs/next/tutorials/compose/page-management.md (hunk 4 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/page-management.md:33]: no... it's an adjective to describe widget, not the `Content` item

File: vuepress/docs/next/tutorials/compose/page-management.md (hunk 8 above)
Comment (Lyd1aCla1r3) [reply vuepress/docs/next/tutorials/compose/page-management.md:70]: i went back and forth... there's no mention of Frames here and "a frame" generalizes the entity

**Review by jyunmitch — APPROVED** [review]
